client/actions/cartActions.js

```diff
@@ -6,15 +6,18 @@ export const getCart = cartItems => ({ type: GET_CART, cartItems });
 // eslint-disable-next-line arrow-parens
 export const fetchCartBySession = id => dispatch => {
```

Comment:
I don't think fetchCartBySession and fetchCartByUser is needed if they're going to be called from the main component, let me know what you think!